src/docs/truffle/how-to/use-the-truffle-dashboard.md

```diff
@@ -76,6 +76,34 @@ module.exports = {
 ```
 
 
+## Decode requests
+
+Truffle Dashboard supports decoding requests for certain RPC calls, allowing you to view a more human-friendly
+representation of the messaging between your computer and the Ethereum network.
```

Comment:
Somewhere in here it might be worth mentioning explicitly that this feature currently only supports Truffle projects (and maybe also that we are keen to add Hardhat support in the near future.)

Or maybe this note belongs in the "Usage with non-Truffle tooling" section below? Or maybe in both places? I don't know... you're the professional writer :)

Reply:
Updated in both places